common/src/main/res/values/colors.xml

```diff
@@ -97,6 +97,8 @@
     <color name="tangerine">#FA9269</color>
 
     <color name="dash_black_0.7">#B30C0C0D</color>
+    <color name="system_green_0.1">#1A3CB878</color>
```

Comment:
This color was also added in the CrowdNode APY PR.  We can fix those merge conflicts later!